doc/rados/operations/pg-repair.rst

```diff
@@ -55,9 +55,9 @@ The "pg repair" command attempts to fix inconsistencies of various kinds. If "pg
 
 "pg repair" will not solve every problem. Ceph does not automatically repair placement groups when inconsistencies are found in them.
 
-The checksum of an object or an omap is not always available. If an object is updated while a scrub is underway, its checksum is marked invalid. Even though an authoritative copy was available when the scrub was underway, that copy's authoritativeness was questionable. Ceph therefore refuses to repair it even if the user runs "pg repair". This is one of the situations in which manual intervention is necessary.
+The checksum of an object or an omap is not always available. Checksums are calculated incrementally. If an object is updated in the middle of a scrub, the write operation involved in the update changes the object and invalidates its checksum. The whole object is not read while recalculating the checksum. So if an object has been updated under these conditions, Ceph doesn't repair it even if the user runs "pg repair". This is one of the situations in which manual intervention is necessary.
```

Comment:
@zdover23 In fact "If a replicated object is updated non-sequentially, the write operation involved..."

Reply:
This change has been added.